README.md

```diff
@@ -29,22 +29,32 @@ when using the `bin/mkosi` shim to configure the python interpreter used
 to execute mkosi. The shim can be symlinked to e.g. `/usr/local/bin` to
 make it accessible from the `PATH`.
 
+```shell
+git clone https://github.com/systemd/mkosi
+ln -s $PWD/mkosi/bin/mkosi /usr/local/bin/mkosi
+mkosi --version
+```
+
 ## Python installation methods
 
 mkosi can also be installed straight from the git repository url using
 `pipx`:
 
 ```shell
 pipx install git+https://github.com/systemd/mkosi.git
+mkosi --version
 ```
 
 which will transparently install mkosi into a Python virtual environment
 and a mkosi binary to `~/.local/bin`. This is, up to the path of the
 virtual environment and the mkosi binary, equivalent to
+
 ```shell
-python -m venv mkosivenv
-mkosivenv/bin/pip install git+https://github.com/systemd/mkosi.git
+python3 -m venv mkosivenv
+mkosivenv/bin/pip3 install git+https://github.com/systemd/mkosi.git
 # the mkosi binary is installed to mkosivenv/bin/mkosi
+source mkosivenv
+mkosi --version
```

Comment:
Do we have to be so specific? Python 2 is EOL, there is no python 2 anymore. ;)

I'd definitely scrap the `3` on the `pip3` though because in a venv all are always only symlinks within the venv to `python` and `pip`. (Apart from that I generally prefer `python -m pip` to `pip`, because it clearly links the used pip to the used python.)

The sourcing can also be scrapped if the mkosi in the venv is called via `mkosivenv/bin/mkosi`.

Reply:
Yes there is no python 2 anymore, but on Ubuntu Jammy python itself doesn't exist, you have to use python3 so it makes sense to use the thing that'll work everywhere.